translations/bnpchs-final-2018-blusukan.html

```diff
@@ -0,0 +1,95 @@
+<!-- Title: Blusukan -->
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Bollyjee presidential election is around the corner. Jelix as the presidential 
+  candidate wants to realize the national commitments by doing <i>blusukan</i> (blusukan 
+  is indonesian word that refer to hands-on interaction with public in order to notice their 
+  necessity).
+</p>
+
+<p style="text-align: justify;">
+  There are a bunch of islands and $N$ province in Bollyjee. The island of two provinces lie on could be the 
+  same or different. Jelix had $M$ information, each information provide two provinces 
+  that lie on same island. If province $A$ and $B$ lie on the same island and province $B$ and $C$ lie 
```

Comment:
```suggestion
  that lie on the same island. If provinces $A$ and $B$ lie on the same island and provinces $B$ and $C$ lie 
```

ada dua provinsi, jadinya pakai plural form